# Get Depth Image

---

sensor_msgs/Image depth_image
